### PR TITLE
Added: postgreSQL documentation

### PR DESCRIPTION
**Pull request by dcagnetta**

### CODE REVIEW

**Comment by ismcagdas** [comment]
Thanks @dcagnetta we will check it.

File: docs/en/Best-Practices/Index.md [hunk 1]
```diff
@@ -24,4 +24,5 @@ Also, this guide is mostly usable for general **application development**.
 * Data Access
   * [Entity Framework Core Integration](Entity-Framework-Core-Integration.md)
   * [MongoDB Integration](MongoDB-Integration.md)
+  * [PostgreSQL Integration](PostgreSQL-Integration.md)
```
Comment (maliming) [inline docs/en/Best-Practices/Index.md]: PostgreSQL Integration should be a sub-item of Entity Framework Core Integration, what do you think?

File: docs/en/Best-Practices/PostgreSQL-Integration.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            {
+                /* This was updated */
+                options.UsePostgreSql();
+				/* This was updated */
+            });
+        }
+    }
+````
+
+### EntityFrameworkCore.DbMigrations Project Update
```
Comment (maliming) [inline docs/en/Best-Practices/PostgreSQL-Integration.md]: Removing the existing migration and then re-adding the migration seems to be simpler and less error-prone.

File: docs/en/Best-Practices/PostgreSQL-Integration.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+### EntityFrameworkCore Project Update
+
+- In `Acme.BookStore.EntityFrameworkCore` project replace package `Volo.Abp.EntityFrameworkCore.SqlServer` with `Volo.Abp.EntityFrameworkCore.PostgreSql` 
+- Update to use PostgreSQL in `BookStoreEntityFrameworkCoreModule`. Example:
+
+````C#
+[DependsOn(
+        //code omitted for brevity
+		
+		/* This was updated from AbpEntityFrameworkCoreSqlServerModule */
```
Comment (maliming) [inline docs/en/Best-Practices/PostgreSQL-Integration.md]: This seems to be more concise, such as: Replace the `AbpEntityFrameworkCoreSqlServerModule `with the `AbpEntityFrameworkCorePostgreSqlModule`.

Similarly: `options.UsePostgreSql()`  : )

**Comment by dcagnetta** [comment]
Hi @maliming, changes made, please could you check? thanks

**Comment by maliming** [comment]
Thank you very much!

**Pull request merged**